test/PHPUnit/Controller/AbstractHttpControllerTestCaseTest.php

```diff
@@ -240,9 +241,17 @@ public function testAssertXpathQueryWithBadXpathUsage(): void
     {
         $this->dispatch('/tests');
 
-        $this->expectError();
+        // symfony/dom-crawler 5.4: InvalidArgumentException
+        // symfony/dom-crawler 6.0: ExpectationFailedException
 
-        $this->assertXpathQuery('form#myform');
+        try {
+            $this->assertXpathQuery('form#myform');
+        } catch (Throwable) {
+            $this->assertTrue(true);
+            return;
+        }
+
+        $this->fail('should not have been called');
```

Comment:
A more descriptive failure message would be helpful. An exception is expected (Assuming DOM Crawler is responsible for throwing this).

```suggestion
        $this->fail('An exception should have been thrown by Symfony DOM Crawler');
```